SimFastTiming/FastTimingCommon/interface/BTLElectronicsSim.h

```diff
@@ -0,0 +1,59 @@
+#ifndef __SimFastTiming_FastTimingCommon_BTLElectronicsSim_h__
+#define __SimFastTiming_FastTimingCommon_BTLElectronicsSim_h__
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "SimDataFormats/TrackingHit/interface/PSimHit.h"
+
+#include "DataFormats/FTLDigi/interface/FTLDigiCollections.h"
+#include "SimFastTiming/FastTimingCommon/interface/FTLDigitizerTypes.h"
+
+namespace ftl = ftl_digitizer;
+
+class BTLElectronicsSim {
```

Comment:
it seems like templating should be used here instead of a BTL class with empty ETL functions and vice versa.